plone/namedfile/scaling.py

```diff
@@ -341,6 +360,15 @@ def getImageSize(self, fieldname=None):
     def guarded_orig_image(self, fieldname):
         return guarded_getattr(self.context, fieldname, None)
 
+    def getRetinaScales(self):
+        try:
+            from Products.CMFPlone.utils import getRetinaScales
+        except ImportError:
```

Comment:
Your other PR (https://github.com/plone/Products.CMFPlone/pull/1611) does implement `getRetinaScales`, why do you want to try/except this import?